Tests/Unit/Module/HardCodedRule.tests.ps1

```diff
@@ -196,6 +196,13 @@ try
                 DscResource  = 'xWinEventLog'
                 IsEnabled    = 'True'
                 LogName      = 'Microsoft-Windows-DnsServer/Analytical'
+            },
+            @{
```

Comment:
I don't believe this test case is correct, the WindowsFeature rule does not have IsEnabled or LogName properties, so when the test case executes it might pass since those properties don't exist for that Rule type and your test case values are $null.

Reply:
Updated